cmd/zt_sync_comparator_test.go

```diff
@@ -619,10 +620,13 @@ func TestDryrunSyncLocaltoFile(t *testing.T) {
 		for i := 0; i < len(msg); i++ {
 			if strings.Contains(msg[i], "DRYRUN: remove") {
 				a.True(strings.Contains(msg[i], dstShareClient.URL()))
-			} else {
+			} else if strings.Contains(msg[i], "DRYRUN: copy") {
```

Comment:
are these changes necessary? Otherwise this PR looks good to me.

Reply:
Yes, we can keep this because now we have added a warn message as part of verification. In this loop we are verifying all the expected logs while performing sync operation from local to file share.